tools/webpack.config.ts

```diff
@@ -33,13 +33,11 @@ export default (env?: string): webpack.Configuration[] => {
                         },
                     ],
                 },
-                // Run the typescript compilier on .ts files before webpack
+                // Transpile .js and .ts files with Babel
                 {
-                    test: /\.tsx?$/,
-                    loader: 'ts-loader',
```

Comment:
This removes type-checking. We should probably add a tool to run `tsc --noEmit` like how we do `./tools/run-mypy`.

Reply:
Yeah, I think that’s the right plan. (We should probably just add `noEmit` to `tsconfig.json`, right?)